src/scripts/index.js

```diff
@@ -1,10 +1,332 @@
+/* import-urile*/
 import '../styles/main.css';
-import Button from './Button';
-import CallToAction from './CallToAction';
+import '../templates/main.html';
 
-const button = new Button(document.querySelector('.button'));
-const cta = new CallToAction(document.querySelector('.button-cta'));
+/* ----------------------- on hover (header) --------------------------*/
 
-button.registerEvents();
-cta.registerEvents();
-cta.changeBgColor();
+const header = document.querySelectorAll('.header');
+
+for (let i = 0; i < header.length; i++) {
+  header[i].addEventListener('mouseenter', enterHeader);
+  header[i].addEventListener('mouseleave', leaveHeader);
+}
+
+/**
+ * Enter in the header section (hover a+)
+ * Make the title invisible and the edit and add options visible
+ * ** PS: elementele din html au si copii de tip text, coloanele adaugate nu,
+ *                    de aici si indecsii respectivi
+*/
+function enterHeader() {
+  const col = event.target.closest('.header').childNodes;
+  if (col[2].nodeName === '#text') {
+    col[1].style.display = 'none';
```

Comment:
If you do end up with [index]-es it still is best to at least hide them with relevant variable names.